Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -40,6 +40,7 @@ public class OrnamentsManager: NSObject {
 
         // Scalebar View
         scalebarView = MapboxScaleBarOrnamentView()
+        scalebarView.isOnRight = options.scaleBar.position == OrnamentPosition.bottomRight || options.scaleBar.position == OrnamentPosition.topRight
```

Comment:
this is confusing to read. It makes sense, but very complex for a ternary case. Can this be put in an actual if statement